### DISCUSSION TITLE
Travail Commun

### DISCUSSION THREAD

**Discussion by Adelineenileda** [elefan-grenoble/gestion-compte#1067] [General]
Bonjour,
Après la mise en place de la caisse nous nous tournons vers la gestion des créneaux. Votre outil a relevé notre attention mais cependant le fait qu'il utilise une version Symphony déprecié nous pose problème. J'ai vu que @samueleyre avait commencé à migrer vers Symphony 4. Est il possible de participer à cette migration ? Comment pourrions nous coopérer ?
Belle journée

**Comment by raphodn** [comment]
Salut Adeline !

Super que tu souhaites aider :)
Je laisse Samuel prendre le relais là-dessus.

Tu es impliquée dans quelle coop du coup ?

**Comment by Adelineenileda** [comment]
Je fais partie de Kissikol à Rouen 😉

**Comment by samueleyre** [comment]
Bonjour Adeline, 

Un coup de main est toujours apprécié ! 
Une fois la mise à jour effectué, on va en particulier avoir besoin d'aide pour tester l'application ( que ce soit manuellement ou en écrivant des tests )

**Comment by Adelineenileda** [comment]
Je gère le Php et Sql mais je connais pas du tout Symphony. Je veux bien écrire des tests mais j'imagine que c'est en lien avec Symphony. Est ce que vous aviez des liens à me conseiller pour comprendre comment faire ça ?

**Reply by samueleyre** [reply]
Il faudrait que tu commences par monter en compétence sur symfony 4, je t'invite déjà à te former sur le framework.  

https://symfony.com/doc/4.0/index.html
https://symfonycasts.com/tracks/symfony4

**Reply by Adelineenileda** [reply]
Je reviens après avoir gérer d'autres priorités...
J'ai trouvé ce tuto et commençais à potasser : https://grafikart.fr/formations/symfony-4-pratique
Parce que l'anglais c'est pas mon truc...

**Reply by samueleyre** [reply]
On est a bientôt finis de stabilisé la version 4 sur la branch develop, tu pourras tester déjà de faire fonctionner cela en local !

**Comment by Adelineenileda** [comment]
D'ailleurs aucune possibilité d'installer sur un hébergement mutualisé ?

**Reply by samueleyre** [reply]
oui c'est possible, @raphodn devrait pouvoir t'aider

**Reply by raphodn** [reply]
alors je n'ai pas vraiment d'expérience en fait ^^
il y a de la doc dans le dossier [/doc](https://github.com/elefan-grenoble/gestion-compte/tree/master/doc).

T'es dans le groupe Telegram ? Certaines coops explorent YunoHost. A l'Elefan ca tourne sur un VPS OVH.

**Reply by Adelineenileda** [reply]
Et j'ai trouvé comment publier en hébergement mutualiser dans ce même tuto :
https://grafikart.fr/tutoriels/symfony-mutualise-1111#autoplay
Faut que je test tout ça !

**Comment by njean42** [comment]
Salut tout le monde,
Je fais partie de Scopéli à Nantes, et suis motivé pour aider au développement du logiciel Elefan.

Pourriez-vous m'ajouter au canal telegram, s'il y a des discussions là-bas aussi ?
J'aimerais y voir plus clair sur qui travaille sur le code, dans quelles coop etc. :slightly_smiling_face: 

Merci !
Nico
